documentation/modules/auxiliary/admin/kerberos/kerberoasting.md

```diff
@@ -0,0 +1,227 @@
+## Vulnerable Application
+
+Any system leveraging kerberos as a means of authentication e.g. Active Directory, MSSQL
+
+## Verification Steps
+
+1. Start msfconsole
+3. Obtain SPNs from your target
+	1. Do: `use auxiliary/gather/ldap_query`
```

Comment:
Are there any benefits/differences/trade-offs in using the ldap_query module directly, versus using the `modules/auxiliary/gather/get_user_spns` module? 👀

Reply:
Oh sweet, I didn't know that module existed. Probably best we direct users to the specific `get_user_spns` module :)

Reply:
@adfoster-r7 `get_user_spns` does seem like the more obvious module to direct users to, although after running a quick test the `ldap_query` module seems to work a bit better (I could be the messing something up) - it also doesn't require the user to input the domain which is nice
```
msfuser@ubuntu:~/git/metasploit-framework$ ./msfconsole -q -x 'use auxiliary/gather/get_user_spns; set rhosts 172.16.199.235; set user DARWIN_CLAY; set pass N0tpassword!; set domain example.com; run'
rhosts => 172.16.199.235
user => DARWIN_CLAY
pass => N0tpassword!
domain => example.com
[*] Running for 172.16.199.235...
[*] Total of records returned 52
[*] No entries found!
[*] Scanned 1 of 1 hosts (100% complete)
[*] Auxiliary module execution completed
```

```
msfuser@ubuntu:~/git/metasploit-framework$ ./msfconsole -q -x 'use auxiliary/gather/ldap_query; set rhosts 172.16.199.235; set BIND_DN  DARWIN_CLAY; set BIND_PW N0tpassword!; set action ENUM_USER_SPNS_KERBEROAST; run'
rhosts => 172.16.199.235
BIND_DN => DARWIN_CLAY
BIND_PW => N0tpassword!
action => ENUM_USER_SPNS_KERBEROAST
[*] Running module against 172.16.199.235
[*] Discovering base DN automatically
[+] 172.16.199.235:389 Discovered base DN: DC=example,DC=com
CN=SHIRLEY_KRAMER OU=Staging OU=Admin DC=example DC=com
=======================================================

 Name                  Attributes
 ----                  ----------
 cn                    SHIRLEY_KRAMER
 samaccountname        SHIRLEY_KRAMER
 serviceprincipalname  CIFS/AWSWWKS1000001

CN=FLORENCE_MOSS OU=Devices OU=OGC OU=Stage DC=example DC=com
=========================================

 Name                  Attributes
 ----                  ----------
 cn                    example.com
 samaccountname        example.com
 serviceprincipalname  http/testService || HTTP/WIN-FDV8QFUMFP7 || HOST/WIN-FDV8QFUMFP7@example.com || HTTP/WIN-FDV8QFUMFP7@example
                       .com

CN=ROMAN_ODONNELL OU=Devices OU=AWS OU=Tier 2 DC=example DC=com
===============================================================
```

Reply:
I've updated this module now - https://github.com/rapid7/metasploit-framework/pull/17490